src/melee.cpp

```diff
@@ -921,6 +927,11 @@ void Character::roll_bash_damage( bool crit, damage_instance &di, bool average,
     float bash_cap = 2 * stat + 2 * skill;
     float bash_mul = 1.0f;
 
+    if( has_trait( trait_KI_STRIKE ) && unarmed ) {
+        /** @EFFECT_UNARMED increases bashing damage with unarmed weapons */
```

Comment:
This is for the doxygen documentation, as is this will be documenting it wrong.
```suggestion
        /** @EFFECT_UNARMED increases bashing damage with unarmed weapons when paired with the Ki Strike trait */
```